jdaviz/configs/mosviz/helper.py

```diff
@@ -191,10 +191,10 @@ def load_data(self, spectra_1d=None, spectra_2d=None, images=None,
             ``Spectrum1D``) that can be read by glue-jupyter. Alternatively,
             can be a string file path.
 
-        images : list of obj, str, or `None`
-            A list of images as translatable container objects
-            (string file path, FITS HDU, FITS HDUList, NDData, or numpy array).
-            Alternatively, can be a string file path. If `None`, no images are displayed.
+        images : list or str
+            A list of images as translatable container objects (e.g.
+            ``CCDData``) that can be read by glue-jupyter. Alternatively,
```

Comment:
No, this stuff shouldn't be here. Something still isn't right.

Reply:
This is probably all coming from a rebase I did yesterday. I am having constant problems with rebasing, like, I can't see the commits that result from merges of main into branch. They only show up on the PR page in github, but not on my local repo. Could it be that my local git command is not properly set?

Reply:
I am not sure what you are doing, but I am seeing a lot merge commits in the history of this PR that should not have been there if you did a _fetch_ + _rebase_ instead of _pull_.

Reply:
I will try to clean up the files manually. Will deal with the commits later.

Reply:
I wonder if it is easier to start over in a new branch from updated upstream/main and you can just re-apply your changes either manually or using cherry-pick.

I tried rebasing this branch and see conflicts like this:

```
Applying: use file name as fallback for object id
Using index info to reconstruct a base tree...
M       jdaviz/configs/mosviz/helper.py
M       jdaviz/configs/mosviz/plugins/parsers.py
Falling back to patching base and 3-way merge...
Auto-merging jdaviz/configs/mosviz/plugins/parsers.py
CONFLICT (content): Merge conflict in jdaviz/configs/mosviz/plugins/parsers.py
Auto-merging jdaviz/configs/mosviz/helper.py
CONFLICT (content): Merge conflict in jdaviz/configs/mosviz/helper.py
error: Failed to merge in the changes.
Patch failed at 0006 use file name as fallback for object id
```

Reply:
Well, one thing I just found: my local repo is not connected to upstream. It's connected to origin. If my origin wasn't properly updated from upstream, then it can explain the mess. Or at least part of it.

Reply:
I have a way to fix this, but I have to bust my fork. Even the `main` branch in my fork is messed up. It appears that code from a branch somehow made into `main` in my fork. I can start from scratch, but this pull request should be closed and I open a new one. We can add a comment to the new one that points to this PR, so our discussion here doesn't get lost. 

This should clear the commit history as well. What do you think?

Reply:
@ibusko , do what you need to do. Good luck!

Reply:
Diffs seem correct now. Commit history has some duplicate entries that I tried to squash, but then it broke the code. I believe that as long as the diffs are correct, merging won't break the product. In case it is important to keep a clean history, I can go ahead and re-start from a new branch.

Reply:
Whoever merges can use the *Squash and Merge* button.